src/components/pages/course-view/CourseViewContainer.jsx

```diff
@@ -1,56 +1,50 @@
 import React, { useState } from 'react';
 import { Card } from 'antd';
+import { ModuleView } from '../index';
 import styled from 'styled-components';
 
+//Component Styles
 const CourseCard = styled(Card)`
   margin-bottom: 4%;
 `;
 
-const courseData = [
-  {
-    courseName: 'Python Basics',
-  },
-  {
-    courseName: 'Problem Solving',
-  },
-  {
-    courseName: 'Time and Space Complexity',
-  },
-];
+const Description = styled.div`
+  margin-bottom: 2%;
+`;
 
 const CourseViewContainer = props => {
   const [isExpanded, setIsExpanded] = useState(false);
+
+  //Props passed from ProgramView Component
+  const { courseName, courseDescription, modules } = props;
+
   return (
     <>
-      <div>
-        {courseData.map(course => (
-          <CourseCard
-            key={Math.random()}
-            title={course.courseName}
-          ></CourseCard>
-        ))}
-        ;
-      </div>
-      <div>
-        <Card title="ViewCourseStream">
-          <span onClick={() => setIsExpanded(!isExpanded)}>
-            {!isExpanded ? 'more' : 'less'}
-            View Course Stream
-          </span>
-        </Card>
-        <Card title="ViewCourseStream">
-          <span onClick={() => setIsExpanded(!isExpanded)}>
-            {!isExpanded ? 'more' : 'less'}
-            View Course Calendar
-          </span>
-        </Card>
-        <Card title="ViewCourseStream">
+      <CourseCard
+        key={Math.random()}
```

Comment:
Again, let's use better keys (not `Math.random()`). `courseId` could be a good key to use here.

Reply:
Will totally make the adjustment, you rock!